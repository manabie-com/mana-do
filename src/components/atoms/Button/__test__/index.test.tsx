import React from 'react';
import {render, screen} from '@testing-library/react';
import {Button} from "../index";

describe('<Button/> render', () => {
        it('should match snapshot', function () {
                const loadingIndicator = render(<Button className="test" color="primary"/>);
                expect(loadingIndicator.container.firstChild).toMatchSnapshot();
        });
        it('should be render with className primary', function () {
                render(<Button/>)
                const buttonElm = screen.queryByTestId('btn primary')
                expect(buttonElm).toBeInTheDocument()
        });
})