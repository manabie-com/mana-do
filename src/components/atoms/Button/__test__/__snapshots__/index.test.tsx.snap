// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Button/> render should match snapshot 1`] = `
<button
  class="test btn btn-primary"
  color="primary"
  data-testid="btn-custom"
/>
`;
